mailpoet/assets/js/src/webpack_admin_expose.js

```diff
@@ -28,3 +28,5 @@ export * as HelpTooltip from 'help-tooltip';
 export * as Hooks from 'hooks.js';
 export * as Listing from 'listing';
 export * as DynamicSegmentsTypes from 'segments/dynamic/types';
+export * as AutomationEditorComponents from 'automation/editor/components';
+export * as AutomationIntegrationComponents from 'automation/integrations/components';
```

Comment:
Thinking about this, `AutomationIntegrationComponents` doesn't make that much sense to me. Maybe, for now, we could move the `FormTokenField` under automation editor components?

Reply:
[5c75571](https://github.com/mailpoet/mailpoet/pull/4604/commits/5c75571d472cd7e1c56050892f99ef1453752ca4)